docs/README.md

```diff
@@ -83,7 +83,7 @@ Open the `src/App.vue` component. Then replace the whole beginning of the file,
     <ais-search-box />
     <ais-results>
       <template scope="{ result }">
-        <h2><ais-highlight :result="result" attribute-name="name" /></h2>
+        <h2 v-html="result._snippetResult.name.value"></h2>
```

Comment:
I changed my mind, I'm going to sanitize every highlighted and snippetted value at results retrieval.
I will try to test the perf impact of doing so and it is, as far as I can see, the easiest way to go about it.